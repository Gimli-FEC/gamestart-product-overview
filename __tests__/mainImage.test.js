import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';

import MainImage from '../client/components/mainImage.jsx';



afterEach(cleanup);

test('renders', () => {
  const { asFragment } = render(<MainImage url={"https://hrr45-fec.s3.us-east-2.amazonaws.com/photos/product/500.jpeg"} />);
  expect(asFragment()).toMatchSnapshot();
});