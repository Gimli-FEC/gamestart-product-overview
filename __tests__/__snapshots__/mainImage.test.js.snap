// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="mainImage__MainContainer-zvt6z9-0 bDNVUL"
    src="https://hrr45-fec.s3.us-east-2.amazonaws.com/photos/product/500.jpeg"
    style="background-position: 0% 0%;"
  >
    <img
      class="mainImage__Image-zvt6z9-1 ccjQdL"
      src="https://hrr45-fec.s3.us-east-2.amazonaws.com/photos/product/500.jpeg"
    />
  </div>
</DocumentFragment>
`;
